test/parallel/test-require-exceptions.js

```diff
@@ -1,16 +1,16 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
 // A module with an error in it should throw
 assert.throws(function() {
   require(common.fixturesDir + '/throws_error');
-});
+}, /blah/);
```

Comment:
Nit: can you please add `^` and `$` respectively at the beginning and the end of the regex?